recipes/carve/build.sh

```diff
@@ -0,0 +1,8 @@
+if [ "$(uname)" == "Darwin" ]; then
+    cmake  ${CMAKE_ARGS} -DCMAKE-INSTALL-PREFIX=${PREFIX} .      
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+    export CXXFLAGS="\$CXXFLAGS -std=c++14"
```

Comment:
What's the reason for the escape here? That could be the source of your issue.
```suggestion
    export CXXFLAGS="$CXXFLAGS -std=c++14"
```

Reply:
Perhaps, I'm trying to switch to cpp 14 somewhat out of desperation as its the only discernable diff with the osx build.